tendrl-node-agent.socket

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=tendrl node agent control socket
+DefaultDependencies=no
+Before=sockets.target
+
+[Socket]
+ListenSequentialPacket=/run/tendrl/message.sock
+SocketMode=0600
```

Comment:
Its not a problem, ListenSequentialPacket is the only option available for AF_UNIX sockets. If the value starts with a slash (/), it is read as file syste socket in the AF_UNIX socket family. please have a look at https://manned.org/systemd.socket/f4f82e36 for further details.